### ISSUE TITLE
Only call `softmax_cross_entropy_with_logits` with named arguments (labels=..., logits=..., ...)

### ISSUE THREAD

**Issue by dengdengdeng123** [echowei/DeepTraffic#21]
softmax_cross_entropy_with_logits参数不对 请问下各位是用什么版本的TensorFlow运行的
